neo/Wallets/Wallet.cs

```diff
@@ -355,12 +356,44 @@ private Transaction MakeTransaction(Snapshot snapshot, TransactionAttribute[] at
                     if (script is null) continue;
                     if (script.IsSignatureContract())
                     {
-                        size += 1 + 66 + script.GetVarSize();
+                        // get Usage/size for this hash (logic can be vastly improved)
+                        int sz = 0;
+                        for(var i=0; i<attributes.Length; i++)
```

Comment:
@igormcoelho this solution is only for cosigners, right?

Reply:
This solution is very bad in fact hahah I'll improve it. In the end, it will work for both native gas transfers (network/system fees) and any invoked nep5 transfers.